src/main/java/spoon/Launcher.java

```diff
@@ -88,6 +90,7 @@
 	protected JSAPResult jsapActualArgs;
 
 	private List<String> processors = new ArrayList<String>();
+	private List<Processor<? extends CtElement>> processorsInstance = new ArrayList<Processor<? extends CtElement>>();
```

Comment:
oh I see. I'm blind :smile: 
Anyway the name is horrible and sounds wrong. "processorInstances" would be the right way to go. But it's not much better.
Why not making it clearer in some way? Call processors -> processorNames and the processorInstances -> processors (the names aren't real processors, just the names)